app/packages/core/src/plugins/SchemaIO/components/DropdownView.tsx

```diff
@@ -8,6 +8,14 @@
 import ChoiceMenuItemBody from "./ChoiceMenuItemBody";
 import FieldWrapper from "./FieldWrapper";
 
+// if we want to support more icons in the future, add them here
+const iconImports: {
+  [key: string]: () => Promise<{ default: React.ComponentType<any> }>;
+} = {
+  MoreVertIcon: () => import("@mui/icons-material/MoreVert"),
+  SettingsIcon: () => import("@mui/icons-material/Settings"),
+};
```

Comment:
@lanzhenw You probably did the manual imports purposefully - just highlighting this suggestion in case you did want to alter your code to do this instead